packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialog.js

```diff
@@ -113,15 +117,18 @@ const styles = {
   },
 };
 
-const mapStateToProps = ({ drillDown, enlargedDialog }) => ({
+const mapStateToProps = ({ global, drillDown, enlargedDialog }) => ({
   ...enlargedDialog,
+  viewConfigs: global.viewConfigs,
   isDrillDownVisible: drillDown.isVisible,
 });
 
 const mapDispatchToProps = dispatch => ({
   onCloseOverlay: () => dispatch(closeEnlargedDialog()),
   onSetDateRange: (startDate, endDate) =>
     dispatch(setEnlargedDashboardDateRange(startDate, endDate)),
+  onSetDrillDownDateRange: (startDate, endDate) =>
```

Comment:
If you move this to `mergeProps`, you could also access the current drill down level by passing it through from `mapStateToProps` - can talk about this if you like